### PR TITLE
Add information about Gerrits WIP feature

### PR DESCRIPTION
**Pull request by DanielSiepmann**
Mention how to allow pushes to WIP state.
Also mention how to configure all pushes to be WIP.

### CODE REVIEW

**Comment by DanielSiepmann** [comment]
@sypets did you already mention that new Gerrit feature somewhere else? In that case we should move all that information to a single place.

File: Documentation/BugfixingAZ/Index.rst [hunk 1]
```diff
@@ -191,6 +191,17 @@ Step by Step Walkthrough
       git push origin HEAD:refs/for/master
 
 
+   .. note::
+      In case you want to push a "Work in progress", use the following instead::
```
Comment (alexanderschnitzler) [inline Documentation/BugfixingAZ/Index.rst]: The following would be a bit more explanatory: `"Work In Progress (WIP)"`.
Mind the uppercase characters.

File: Documentation/BugfixingAZ/Index.rst [hunk 2]
```diff
@@ -191,6 +191,17 @@ Step by Step Walkthrough
...
+
+          git push origin HEAD:refs/for/master%wip
+
+      You can also configure Gerrit to always mark your pushed as WIP. Therefore
```
Comment (sgrossberndt) [inline Documentation/BugfixingAZ/Index.rst]: pushed -> pushes

File: Documentation/CheatSheets/Git.rst [hunk 3]
```diff
@@ -59,18 +59,20 @@ Push changes to remote master on gerrit (default method)::
...
+===========================
 
-Push changes to remote master on gerrit as **draft**::
+In case you want to push a "Work in progress", use the following instead::
 
-   git push origin HEAD:refs/drafts/master
+      git push origin HEAD:refs/for/master%wip
 
+You can also configure Gerrit to always mark your pushed as WIP. Therefore
+head over to https://review.typo3.org/settings/ and configure "Set new
```
[the remark this answers is not in the document]
Comment (sgrossberndt) [reply Documentation/CheatSheets/Git.rst]: This has not been resolved yet: `mark your pushed as WIP` must be `mark your pushes as WIP`

**Comment by sypets** [comment]
@DanielSiepmann I have not made changes in contribution guide recently.

Generally, I'd be happy to review, but I am currently not well informed about changes. So, go ahead and do it. Should be fine if Stephan agrees.

**Comment by sgrossberndt** [comment]
There still is an issue:
https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContributionWorkflow/pull/123/files#r296558456
Otherwise its fine in my opinion.

**Comment by DanielSiepmann** [comment]
@sgrossberndt The link doesn't show me any issue, what exactly is the issue that I should fix?

**Comment by sgrossberndt** [comment]
`mark your pushed as WIP` must be `mark your pushes as WIP` instead.

**Comment by sypets** [comment]
I fixed what @sgrossberndt pointed out. Please look at final version. Thanks. :thumbsup:

**Pull request merged**